### DISCUSSION TITLE
(audacity) Update dependencies for clarity

### DISCUSSION THREAD

**Discussion by TheNavyBear** [chocolatey-community/chocolatey-packages#1902] [Ideas]
Howdy,

Is there a way to update the Audacity dependency on vcredist2015 to vcredist140? I know vcredist2015 just installs vcredist140 anyway, but then you have vcredist2015 and vcredist140 both in the app list, making it a bit cluttered. It's not a functional issue, just a cosmetic one. Thank you for any help you can provide, and a big thank you for maintaining this package for the community.

**Comment by TheCakeIsNaOH** [comment]
I don't think there would be any harm in updating the dependency to install vcredist140 directly. Pull requests are welcome on this repository.

However, even if the dependency was updated, it would not automatically uninstall the vcredist2015 package, it would have to be uninstalled with a manual `choco uninstall` run.

**Reply by TheNavyBear** [reply]
Understood. Thank you for the info. I would make a PR if I knew how, here's hoping someone who knows how will kindly do it.

**Reply by slycordinator** [reply]
Also, it could be preferable to have it depend on vcredist2017, since vcredist2017 is apparently an updated drop-in replacement for vcredist2015.

**Comment by AdmiringWorm** [comment]
Before anyone thinks about changing the dependency, it needs to be verified that this dependency is really needed. I do not see any mention of it on their webpage anymore, but I believe it did when the dependency was initially added (this should be done every time a dependency is changed or updated).

I have no problems with someone putting up a pull request to change the dependency to `vcredist140` if it is still needed.
